### DISCUSSION TITLE
[@types/frida-gum] Module error: node_modules/@types/frida-gum/index.d.ts is not a module.

### DISCUSSION THREAD

**Discussion by Bistaff** [DefinitelyTyped/DefinitelyTyped#69118] [Issues with a @types package]
Hi all,

I created a project with typescript. I installed @types/frida-gum with npm and when I tried to import it on my file `import * as fridagum from '@types/frida-gum';` I got this error:

`node_modules/@types/frida-gum/index.d.ts is not a module`.

Can anyone help me?

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
If you're trying to include the types defined in `@types/frida-gum`, then you need to use a [triple-slash directive](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-). `@types` packages aren't actually modules on their own.

You should be able to include the types from `@types/frida-gum` with a comment like this in your source:

```typescript
/// <reference types="frida-gum" />
```
